### DISCUSSION TITLE
Daily Standup - Monday, May 29

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#196] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
-

**Shoutouts**:
- @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
- Great Hackathon 

**What are your priorities for the next 24 hours?**:
- Check for bugs, work on architecture doc

**Blockers**:
- Really tired

**Shoutouts**:
- @Int-Angel he is a rifado

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
- Finished implementing #195, a menu for uploading and interacting with our data sources.

**What are your priorities for the next 24 hours?**:
- Work on the architecture documentation.

**Blockers**:
- I am wrapping up with my current job, I have been a little busy.

**Shoutouts**:
- @Int-Angel for working on the backend code for creating charts within our boards
- @DavBot02 for finishing so many tasks in so little time
- @Jhoscar22 for building more useful datasources.
- @paotg123 for being so proactive with our documentation
